src/use_case/add_project/AddProjectInteractor.java

```diff
@@ -87,7 +87,8 @@ public void execute(AddProjectInputData addProjectInputData) {
                 String[] taskAttributes = task.split("~");
                 tasksString.append(taskAttributes[0]).append('`');
                 tasksString.append(taskAttributes[1]).append('`');
-                tasksString.append(taskAttributes[2]).append("|uwu|");
+                tasksString.append(taskAttributes[2]).append('`');
```

Comment:
I'm assuming this change does not have to be reflected in the Entities? Just in the frontend? Because tasksString is only used for output data and is not used to modify entities.

Reply:
When new tasks are created, their statuses are automatically set to false. I'm assuming this change is only so that the front end has access to the status of the tasks.